linode_api4/groups/linode.py

```diff
@@ -303,6 +306,15 @@ def instance_create(
             fw = kwargs.pop("firewall")
             kwargs["firewall_id"] = fw.id if isinstance(fw, Firewall) else fw
 
+        if "interfaces" in kwargs:
+            interfaces = kwargs.pop("interfaces")
+            param_interfaces = []
+            for interface in interfaces:
+                if isinstance(interface, ConfigInterface):
+                    interface = interface._serialize()
+                param_interfaces.append(interface)
+            kwargs["interfaces"] = param_interfaces
```

Comment:
```suggestion
            kwargs["interfaces"] = [i._serialize() if isinstance(i, ConfigInterface) else i for i in kwargs["interfaces"]]
```

One line alternative, just an optional suggestion